tests/requirements-libraries.txt

```diff
@@ -30,7 +28,9 @@ babel==2.8.0
 future==0.18.2
 gevent==20.9.0
 pygments==2.7.1
-PySide2==5.14.0
+pyside2==5.15.1
+pyqt5==5.15.1
+pyqtwebengine==5.15.1
```

Comment:
How is this realted to Python 3.0?

Reply:
I am not sure I understand your comment 🤔

Reply:
This Qt stuff is fine - it's due to test failures which I'm pretty sure no longer are there in newer versions of Qt.